packages/nodes-base/nodes/Google/Calendar/GoogleCalendar.node.ts

```diff
@@ -38,15 +28,15 @@ export class GoogleCalendar implements INodeType {
 		description: 'Consume Google Calendar API.',
 		defaults: {
 			name: 'Google Calendar',
-			color: '#3E87E4',
+			color: '#3E87E4'
 		},
 		inputs: ['main'],
 		outputs: ['main'],
 		credentials: [
 			{
 				name: 'googleCalendarOAuth2Api',
```

Comment:
Is this an editor config thing? Does n8n have a suggested config? It would be easier to review without the formatting changes.